### DISCUSSION TITLE
Project Boards

### DISCUSSION THREAD

**Discussion by tvvocold** [community/community#4052] [Mobile] [answered]
Plz consider moving [the Project Boards feature](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/about-project-boards) to GitHub iOS App as well, thx.

**Comment by aconsuegra** [comment, marked as answer]
Thank you so much for the request! We are tracking being able to manage projects internally and I added your note to our issue. In the meantime, please let us know if you have any other bugs or feature requests. We really appreciate it!

And please, upvote the existing feature request for [Projects here](https://github.com/github/feedback/discussions/3023).
